docs/fork-clone-pullrequest.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Fork, Clone and Pullrequest
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /fork-clone-pullrequest.html
+summary:
+---
+
+Fork/Clone 
```

Comment:
I would prefer to keep documentation about talk exclusively about Talk. We can include a reference on how to create PR's, but not on our own docs.